src/Administration/Resources/app/administration/src/app/component/media/sw-media-base-item/sw-media-base-item.scss

```diff
@@ -122,7 +122,7 @@ $sw-media-base-item-preview-background:      $color-white;
 
             border: 1px solid transparent;
             line-height: 27px;
-            direction: rtl;
+            direction: ltr;
```

Comment:
do we need this at all?

Reply:
@mitelg Don't know if it is needed but maybe someone put it here for a reason. What do you want me to do with it?

Reply:
if you delete it completely, is the result still the same?

Reply:
@mitelg Yes, it seems to work ok when I delete it.